e2e/miniconstellation/test-remote.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+#
+# This script expects:
+# Constellation CLI @ $PWD/constellation
+# kubectl @ PATH
+
+set -euo pipefail
+
+debug_info() {
+  arg=$?
+
+  echo "Getting debug info"
+
+  sha256sum ./*.raw
+
+  ls -lisah
+
+  cat ./constellation-conf.yaml
+
+  exit "${arg}"
+}
+
+trap debug_info ERR
+
+echo "::group::Run E2E Test"
+mkdir constellation_workspace
+cd constellation_workspace
+cp ../constellation .
+chmod u+x constellation
+
+# wait for docker to come up
+echo "Waiting for docker service to be active..."
+
+# Wait at most 5min
+count=0
+until systemctl is-active docker || [[ ${count} -eq 30 ]]; do
+  sleep 10
+  count=$((count + 1))
+done
+
+if [[ ${count} -eq 30 ]]; then
+  echo "Docker service did not come up in time."
+  exit 1
+fi
+
+echo "Done waiting."
+
+./constellation mini up --debug
+
+export KUBECONFIG="${PWD}/constellation-admin.conf"
+
+# Wait for nodes to actually show up in K8s
+count=0
+until kubectl wait --for=condition=Ready --timeout=2s nodes control-plane-0 || [[ ${count} -eq 30 ]]; do
+  sleep 10
+  count=$((count + 1))
+done
+
+count=0
+until kubectl wait --for=condition=Ready --timeout=2s nodes worker-0 || [[ ${count} -eq 30 ]]; do
+  sleep 10
+  count=$((count + 1))
+done
```

Comment:
The output of this part could be improved a little, it currently reads

```
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
Error from server (NotFound): nodes "worker-0" not found
error: timed out waiting for the condition on nodes/worker-0
error: timed out waiting for the condition on nodes/worker-0
error: timed out waiting for the condition on nodes/worker-0
```
confused me first. Maybe pipe /dev/null and add some user friendly output.

Reply:
Would be also nice to add some indention to various pieces of output to get it more structured and readable.

Reply:
Can you give an example for your second suggestion?